addons/web/static/tests/views/fields/one2many_field_tests.js

```diff
@@ -5444,10 +5458,11 @@ QUnit.module("Fields", (hooks) => {
                 ".o_selected_row .o_field_widget[name=display_name] input",
                 "new val"
             );
+            await editInput(target, ".o_selected_row .o_field_widget[name=foo] input", "new foo");
```

Comment:
is this line necessary?

Reply:
I want to check if the commands it s correctly send.

Reply:
We don t care the new foo, we want to check the previous change.

Reply:
I want to check if the update command for "new val" are in the relationalField. And we don t want the new foo (Check message from rco)